pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -17,13 +17,27 @@ limitations under the License.
 package enqueue
 
 import (
+	"fmt"
+	"strconv"
+	"strings"
+
 	"github.com/golang/glog"
 
 	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/conf"
 	"volcano.sh/volcano/pkg/scheduler/framework"
 	"volcano.sh/volcano/pkg/scheduler/util"
 )
 
+// DefaultEnqueueActionKey
+const DefaultEnqueueActionKey = "enqueue-action-idleres-mul"
```

Comment:
I donot like this name `DefaultEnqueueActionKey `, it is meaningless.

Reply:
`enqueue-action-idleres-mul` is too long @k82cn wdyt?

Reply:
yes, we need to have a better name.